packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -1503,6 +1503,7 @@ class _DialogRoute<T> extends PopupRoute<T> {
     return Semantics(
       child: _pageBuilder(context, animation, secondaryAnimation),
       scopesRoute: true,
+      focusable: true,
```

Comment:
Why's a route hard-coded to be focusable?

Reply:
Good point.  Not needed anymore, I don't think, since the Focus will provide the semantics if the route has anything focusable on it.